docs/Migrating.md

```diff
@@ -1,5 +1,26 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.8 to v0.9
+
+### Important Changes
+* We have changed the way reward signals (including Curiosity) are defined in the
+`trainer_config.yaml`.
```

Comment:
Maybe instead of saying "it has changed in the trainer_config.yaml" (which is one of the yaml file that doesn't have any gail configured), say something like "we have changed the way reward signals (..) are defined in the configuration file, you can refer to gail_config.yaml as an example".

Reply:
Please ignore my previous comment. I just realized that this isn't explaining anything regarding the new features, just how to migrate from using the old features.